develdocs.phtml

```diff
@@ -14,7 +14,7 @@ include("externals/menu.phtml");
 
 <p class="justify"><?=$description?></p>
 
-<p class="justify"><?php echo T_("Just as the <span class=\"gnucash\">GnuCash</span> application has been developed over time by volunteers, so too has the documentation been written by volunteer developers and users to help others learn how <span class=\"gnucash\">GnuCash</span> works. This collaborative effort has resulted in the creation of two major documents, the <b>Help Manual</b> and the <b>Tutorial and Concepts Guide</b>.");?></p>
+<p class="justify"><?php echo T_("Just as <span class=\"gnucash\">GnuCash</span> has taken form over time, so too has the documentation to help others learn how it works. There are two major documents, the <b>Help Manual</b> and the <b>Tutorial and Concepts Guide</b>.");?></p>
```

Comment:
This changes the original text to a new meaning. The original aims to bring out the volunteer nature of the project; this change eliminate that entirely--a change in intent with which I do not agree.

Reply:
The whole sentence (now) 
> Just as <span class=\"gnucash\">GnuCash</span> has taken form over time, so too has the documentation to help others learn how it works.

Doesn't do anything, if "volunteer" is moved to the next sentence, describing the two components.